keyboards/dztech/dz60rgb_ansi/keymaps/bingocaller/config.h

```diff
@@ -0,0 +1,9 @@
+#define MOUSEKEY_DELAY 0 // default is 300ms
```

Comment:
```suggestion
#pragma once

#define MOUSEKEY_DELAY 0 // default is 300ms
```

Reply:
Thank you for reviewing, @fauxpark!

It wasn't entirely clear to me whether or not `#pragma once` was needed for every header file or only if needing old-style include guards (I don't know what either of those things mean, because I'm not familiar with C). I'm referring to this part of [the C coding conventions documentation](https://docs.qmk.fm/#/coding_conventions_c):

> We use `#pragma once` at the start of header files rather than old-style include guards (`#ifndef THIS_FILE_H`, `#define THIS_FILE_H`, …, `#endif`)

Perhaps this could be reworded to be more clear?

Reply:
Wikipedia has a good rundown on the use of `#pragma once` vs include guards: https://en.wikipedia.org/wiki/Pragma_once

Because the `#include` preprocessor statement is basically a glorified copy-paste, both of these ensure that if for example a .c file and one of the headers it brings in both include the same header, it is only "pasted" once to avoid multiple definition errors.

Reply:
Very cool. Thank you for clarifying! 😊